amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -378,3 +396,22 @@ type CurrentVersion @model {
   key: String! @index(name: "byKey", queryField: "getCurrentVersionByKey")
   version: Int!
 }
+
+type ColonyAction @model {
+  id: ID!
+  colonyId: ID! @index(name: "byColony", queryField: "getActionsByColony")
+  colony: Colony! @belongsTo(fields: ["colonyId"])
+  type: ColonyActionType!
+  blockNumber: Int!
+  transactionHash: String!
+  initiatorAddress: ID
+  initiator: User @hasOne(fields: ["initiatorAddress"])
+  recipientAddress: ID
+  recipient: User @hasOne(fields: ["recipientAddress"])
```

Comment:
As discussed in a call with Raul and Will, this will be updated to support different types of initiator/recipient (e.g. Colony, Extension)